Formula/percona-server.rb

```diff
@@ -65,14 +71,15 @@ class PerconaServer < Formula
     sha256 "6709edb2393000bd89acf2d86ad0876bde3b84f46884d3cba7463cd346234f6f"
   end
 
-  # Fix build on Monterey.
-  # Remove with the next version.
-  patch do
-    url "https://github.com/percona/percona-server/commit/e5b2df737987ce72d285b68fb802cf5b1a15f843.patch?full_index=1"
-    sha256 "6b529dc9e756d137429c36625aafb309298510c46705d0bef71b9265d53e6d80"
-  end
-
   def install
+    # Fix mysqlrouter_passwd RPATH to link to metadata_cache.so
+    inreplace "router/src/http/src/CMakeLists.txt",
+              "ADD_INSTALL_RPATH(mysqlrouter_passwd \"${ROUTER_INSTALL_RPATH}\")",
+              "\\0\nADD_INSTALL_RPATH(mysqlrouter_passwd \"${RPATH_ORIGIN}/../${ROUTER_INSTALL_PLUGINDIR}\")"
```

Comment:
If anyone has Oracle support account, may be worth checking what is official solution at: https://support.oracle.com/knowledge/Oracle%20Database%20Products/2745700_1.html

I haven't created account yet, so haven't seen what is recommended by Oracle.

As note, this also impacts macOS `mysqlrouter_passwd`, but CI doesn't detect it due to variable-referenced `@rpath`.

Also should impact `mysql` formula.

Reply:
Was it broken when running it? Looks like mysql formula just has: https://github.com/Homebrew/homebrew-core/blob/ad269549441fee3b107d4c8d335b0f39fb39cbab/Formula/mysql.rb#L42

Reply:
Yes, it fails in current version:
```console
% mysqlrouter_passwd
dyld[77066]: Library not loaded: @rpath/metadata_cache.so
  Referenced from: /usr/local/Cellar/percona-server/8.0.26-17/bin/mysqlrouter_passwd
  Reason: tried: '/usr/local/Cellar/percona-server/8.0.26-17/bin/../metadata_cache.so' (no such file), '/usr/local/Cellar/percona-server/8.0.26-17/bin/metadata_cache.so' (no such file), '/usr/local/Cellar/percona-server/8.0.26-17/bin/../lib/metadata_cache.so' (no such file), '/usr/local/Cellar/percona-server/8.0.26-17/bin/../metadata_cache.so' (no such file), '/usr/local/Cellar/percona-server/8.0.26-17/bin/metadata_cache.so' (no such file), '/usr/local/Cellar/percona-server/8.0.26-17/bin/../lib/metadata_cache.so' (no such file), '/usr/local/lib/metadata_cache.so' (no such file), '/usr/lib/metadata_cache.so' (no such file)
zsh: abort      mysqlrouter_passwd
```